ui/src/components/head/index.js

```diff
@@ -10,7 +10,7 @@ export default (props: any, css: string) =>
     `<head lang="en" data-page-path="/uk">
```

Comment:
should this be named something else than `default` since it represents head for the 404 page